### ISSUE TITLE
[Feature request] explicit option to exclude the DB

### ISSUE THREAD

**Issue by dougle03** [sabeechen/hassio-google-drive-backup#180]
My backups are huge at over 600mb each. This is largely  due to the size of the HA sqlDB file.
Any chance of a feature to exclude file types from a backup, ora switch for this.
Note that my recorder is already set to purge 1 day, but I have a lot of sensors and not too bothered if the DB is not restored from a backup.
Thanks,

**Comment by sabeechen** [comment]
I like the idea of this but I'm not sure if I have a way to implement it because:
* Home Assistant allows a truly remarkable number of ways to configure the database
* Snapshot can only exclude a whole folder, not a specific file (this is a Supervisor limitation)

I totally agree that the database is a big pain point for a lot of people.  IMHO its an *extremely* poorly thought out part of Home Assistant and often the first problem power users run into.

What I've seen some do (haven't tried it myself) is store their database in a folder like /share and then exclude that folder.  You should be able to do it by adding something like this to your configuration.yaml:
```yaml
recorder:
  db_url: sqlite:////share/database/new_database.db
```

**Comment by dougle03** [comment]
Cracking idea, I'll move the DB to a new folder and then exclude that folder... Thanks

**Comment by digiltd** [comment]
Perhaps a way to stop the db before making the snapshot? I had big problems recently when my MariaDB backup was corrupt. Turns out this is a common problem when the snapshot is being made whilst something is writing to it.

**Comment by sabeechen** [comment]
I'm closing out old issues that I'm considering resolved, but feel free to reopen this if I missed something.

BTW, I just released v0.101.3 which adds the ability to stop specific addons while a snapshot is being taken.  You can use that to stop the MariaDB addon as you suggested.
